eo-maven-plugin/src/test/java/org/eolang/maven/WalkTest.java

```diff
@@ -23,14 +23,21 @@
 final class WalkTest {
 
     @Test
-    void findsFiles(@Mktmp final Path temp) throws Exception {
-        new HmBase(temp).save("", Paths.get("foo/hello/0.1/EObar/x.bin"));
-        new HmBase(temp).save("", Paths.get("EOxxx/bar"));
+    void findsFilesMatchingGlobPattern(@Mktmp final Path temp) throws Exception {
+        final String nonmatch = "foo/hello/0.1/EObar/x.bin";
```

Comment:
@a1kuat `nonmatch` and `match` variables seems to use only once, I think we can leave them inlined